CHANGELOG.md

```diff
@@ -5,21 +5,40 @@ FEATURES:
   The default behavior for this feature is `enabled: true`.
   See [https://www.consul.io/docs/k8s/health](https://www.consul.io/docs/k8s/health) for more information.
   In order to enable this feature for existing installations it is required to restart all connect injected deployments so that they are re-injected.
-  Until this is done health checks for these deployments will not be synced to Consul.
+  Until this is done, health checks for these deployments will not be synced to Consul.
 
   **It is recommended to enable TLS with this setting enabled because it requires making calls to Consul clients across the cluster.
     Without TLS enabled, these calls could leak ACL tokens should the cluster network become compromised.**
+* Support for custom resource definitions (CRDs) is now generally available.
+  CRDs require Consul >= 1.8.4. If you wish to use `ServiceIntentions`
+  custom resources then this requires Consul >= 1.9.0 (which is still in beta as of this release).
+
+  To enable, set `controller.enabled: true` in your Helm configuration:
+
+  ```yaml
+  controller:
```

Comment:
should we mention global here?

Reply:
Do you mean like:
```yaml
global:
  name: consul
controller:
  enabled: true
```

`controller` isn't under the `global` key.

Reply:
ah okay then nevermind :)